pulumi/src/volumeSnapshotClient.ts

```diff
@@ -0,0 +1,48 @@
+import * as k8sClient from '@kubernetes/client-node'
+
+export interface VolumeSnapshot extends Required<k8sClient.KubernetesObject> {
```

Comment:
Would be nice to leverage existing types over copy pasta, but the project structure has become a little jank. I am going to lift all of the node config to top level so we can structure this a bit better and leverage eslint/prettier throughout as intended

Reply:
I have that as a 4th step or so - ideally to pull in all Pulumi stuff into a single tree, separate from the coinstack implementations

Reply:
yeah full separation as a later step, have a poc of just pulling node workspaces up one level and should make it quite a bit cleaner

Reply:
I'll keep it in here for now, we'll need to marry the reaper module with the rest to share the import